lib/sidekiq/cron/schedule_loader.rb

```diff
@@ -0,0 +1,14 @@
+require "sidekiq"
+require "sidekiq/cron/job"
+
+if Sidekiq.server?
+  Sidekiq.configure_server do |config|
+    schedule_file = config.options[:cron_schedule_file] || "config/schedule.yml"
+
+    if File.exist?(schedule_file)
+      config.on(:startup) do
+        Sidekiq::Cron::Job.load_from_hash YAML.load_file(schedule_file)
```

Comment:
I myself have this question, why not use `!`? 
I believe whenever the schedule file is updated and some jobs are removed, they need to be removed from schedule too. 

```suggestion
        Sidekiq::Cron::Job.load_from_hash! YAML.load_file(schedule_file)
```

Reply:
You can skip automatic loader by renaming your `schedule.yml` file, or by setting `cron_schedule_file` path to non-existing file. And then load you schedule files manually.